Gems/Atom/RHI/Vulkan/Code/Include/Atom/RHI/VkAllocator.h

```diff
@@ -0,0 +1,31 @@
+/*

```

Comment:
This header will need to go to RHI.Reflect folder as we pull that into OpenXrVk gem. Also, are you working on a PR to update the OpenXrVk gem as well (to use this allocator)?

Reply:
The header file is moved to `Gems/Atom/RHI/Vulkan/Code/Include/Atom/RHI.Reflect`.
I will see to `OpenXrVk` after #14526 is merged.